### PR TITLE
Remove section-container margins imposed by social block

### PR DESCRIPTION
**Pull request by sdmcraft**
Test URLs:
- Before: https://main--sunstar--hlxsites.hlx.live/newsroom/update-of-sunstars-esg-activities-for-2023
- After: https://i-309b--sunstar--hlxsites.hlx.live/newsroom/update-of-sunstars-esg-activities-for-2023

- [ ]  If you are adding a new block or a variation to an existing block, has it been added in the [block-inventory](https://adobe.sharepoint.com/:w:/r/sites/HelixProjects/Shared%20Documents/sites/sunstar/sunstar-engineering/_drafts/block-inventory.docx?d=w0e883b961c9f4401bae37da23ace83ec&csf=1&web=1&e=KcfAim) ?

### CODE REVIEW

**Review by ehrro — CHANGES_REQUESTED** [review]
Hello @sdmcraft ,

It breaks the layout on the article page. The layout is no longer the right size.
image

Let's talk about this tomorrow during our meeting.

**Comment by sdmcraft** [comment]
Hi @ehrro , @jindaliiita , I made a few adjustments. Can you please take a look at https://i-309b--sunstar--hlxsites.hlx.page/newsroom/world-dental-hygienist-awards-ceremony-australia if it looks ok?

File: styles/styles.css [hunk 1]
```diff
@@ -1094,7 +1091,7 @@ main .block.centered > div {
   }
 
   /* section - narrow */
-  main .section.narrow {
+  main .section.narrow > .section-container {
```
Comment (jindaliiita) [inline styles/styles.css:1094]: Why this changes, this will impact the layout

**Comment by jindaliiita** [comment]
I am wondering why this change is required

**Comment by sdmcraft** [comment]
> I am wondering why this change is required

Please take a look at [this conversation](https://adobe-dx-support.slack.com/archives/C051MV16P0D/p1699432240969429). Basically the idea is to have the ability to apply section variants at section-container level.

File: styles/styles.css (hunk 1 above)
Comment (sdmcraft) [reply styles/styles.css:1094]: > Why this changes, this will impact the layout

The variants such as `narrow, narrorwer` etc, should apply to `section-container`. The [context is here](https://cq-dev.slack.com/archives/C058KBHLBGR/p1699440902822529). I made a few adjustments to eliminate the impact on layout. Please take a look.

**Comment by sdmcraft** [comment]
> Hello @sdmcraft there is still the same layout shift on the article pages.

@ehrro , I made a few adjustments to make the layout same as before. Please take a look.

As you'd already know ([we discussed about this here](https://adobe-dx-support.slack.com/archives/C051MV16P0D/p1699432240969429)), the reason I am after this change is to get the ability to specify section level styles such as background colors, images etc. which right now isn't possible. For e.g. let's say you want to apply a blue background or an image to the entire section which is styled as `narrow`, there's isn't a way to do that because `narrow` gets applied `section` instead of `section-container`.  Same holds true for any section variant like `narrower, padded` etc.  which restricts the section width etc. instead of the `section-container`.

File: blocks/social/social.css [hunk 2]
```diff
@@ -222,4 +224,9 @@ main .section.social-container a.category-title {
   .block.social {
     right: -10rem;
   }
+
+  main .section.narrow.social-container > .section-container {
```
Comment (dnbute) [inline blocks/social/social.css:228]: I think we need a combination of `margin-right` and `margin-left` because at certain resolutions the page shifts to the left
Screenshot 2023-11-13 at 10 40 26

Comment (sdmcraft) [reply blocks/social/social.css]: > I think we need a combination of `margin-right` and `margin-left` because at certain resolutions the page shifts to the left 

Thanks for pointing this out. This should be addressed now. Please check.

Comment (dnbute) [reply blocks/social/social.css:229]:
```
margin-left: auto;
margin-right: auto;
```
seems to do the trick (the page ever so slightly shifts to the right, but it seems acceptable)

**Comment by sdmcraft** [comment]
@ehrro , please let me know if this is good to go. You may notice a few visual differences indicated by the visual tests. If they are acceptable, we can proceed. For the low PSI reported, its being tracked here #355

**Review by ehrro — APPROVED** [review]

**Comment by jindaliiita** [comment]
@sdmcraft This PR also impacted page **healthy-thinking/how-to-develop-a-healthy-habit**
Correct one - https://minor-plugin-change--sunstar--hlxsites.hlx.live/healthy-thinking/how-to-develop-a-healthy-habit
After change - https://main--sunstar--hlxsites.hlx.live/healthy-thinking/how-to-develop-a-healthy-habit

Check tags section
